react/features/welcome/components/WelcomePage.web.js

```diff
@@ -360,6 +341,16 @@ class WelcomePage extends AbstractWelcomePage {
     }
 
     /**
+    * Handles updating roomname.
+    *
+    * @private
+    * @returns {void}
+    */
+    _onUpdateRoomname() {
+        this._updateRoomname();
+    }
+
+    /**
```

Comment:
I personally removed the method _onUpdateRoomname and its uses. You've brought back the method but not its uses.